tests/unit/DateUtilsTest.js

```diff
@@ -132,12 +132,10 @@ describe('DateUtils', () => {
 
     describe('Date Comparison Functions', () => {
         const today = new Date();
-        const tomorrow = addDays(today, 1);
-        const yesterday = subDays(today, 1);
 
         const todayInTimezone = utcToZonedTime(today, timezone);
-        const tomorrowInTimezone = utcToZonedTime(tomorrow, timezone);
-        const yesterdayInTimezone = utcToZonedTime(yesterday, timezone);
+        const tomorrowInTimezone = addDays(todayInTimezone, 1);
+        const yesterdayInTimezone = subDays(todayInTimezone, 1);
```

Comment:
FYI this was the main test fix, but for some reason I also had to update `isYesterday` (in DateUtils) to get this to work - so then I updated `isToday` and `isTomorrow` to look consistent